modules/ROOT/pages/creating-a-new-look-feel.adoc

```diff
@@ -13,10 +13,6 @@ A Look & Feel is defined in a set of files. You can set the Look & Feel for the
 
 == Introduction
 
-To create a new Look & Feel in Bonita Portal, xref:managing-look-feel.adoc[export the default Look & Feel] and modify the relevant components.
-You can either edit the files directly, or you can use a plugin and update the Look & Feel in a browser.
-When the new Look & Feel is ready, xref:managing-look-feel.adoc[import and apply the new Look & Feel].
-
 There are two types of page in the Portal: pages developed with GWT, as in earlier versions, and new pages, such as the Case monitoring page, that have been developed using Angular.
```

Comment:
will this part be handled by @baptistemesta ?

Reply:
I only deleted pages that were mentioned in the [Runtime-490](https://bonitasoft.atlassian.net/browse/RUNTIME-490) ticket.

Reply:
Since manage-look-feel is one of them, that's why I deleted it